huxley/core/admin/committee_feedback.py

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2011-2015 Berkeley Model United Nations. All rights reserved.
+# Use of this source code is governed by a BSD License (see LICENSE).
+
+from django.conf.urls import url
+from django.contrib import admin
+from django.core.urlresolvers import reverse
+from django.http import HttpResponseRedirect
+
+from huxley.core.models import CommitteeFeedback
+
+
+class CommitteeFeedbackAdmin(admin.ModelAdmin):
```

Comment:
This file is unnecessary I believe

Reply:
The default admins should work just fine; you can omit the second argument to register in the __init__ file

Reply:
Done